2024-09-15 08:51:27,382 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:52:27,486 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:53:28,136 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:54:28,603 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:55:29,116 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:56:29,913 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:57:30,330 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:58:30,883 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 08:59:31,311 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:00:31,413 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:01:32,737 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:02:33,758 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:03:34,280 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:04:34,377 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:05:35,643 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:06:36,660 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:07:36,758 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:08:37,704 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:09:37,782 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:10:38,708 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:11:39,282 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:12:39,388 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:13:39,892 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:14:40,472 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:15:41,499 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:16:41,603 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:17:42,536 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:18:42,615 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:19:43,634 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:20:43,738 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:21:44,956 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
2024-09-15 09:22:45,056 ERROR scheduler Exception in Enqueue Events for Site greenField.local
Traceback (most recent call last):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 117, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 130, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 841, in get_single_value
    ).run()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 211, in sql
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/database.py", line 117, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/joric/wsl-frappe/frappe-bench/frappe-bench/env/lib/python3.10/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_d55f8d240b1dc9c2'@'localhost' (using password: YES)")
